backend/libraries/group_chat_core/src/lib.rs

```diff
@@ -1572,7 +1571,6 @@ impl GroupChatCore {
 
         if public_changed || message_visbility_changed {
             let event = GroupVisibilityChanged {
-                now_public: self.is_public.value,
```

Comment:
Let's change the frontend over in time for the next release

Reply:
Actually we probably want the frontend out before groups are finished

Reply:
Yes just spoken to Julian about it